src/platform/forms-system/src/js/widgets/TextWidget.jsx

```diff
@@ -34,7 +34,9 @@ export default function TextWidget(props) {
     onChange: event =>
       props.onChange(event.target.value ? event.target.value : undefined),
     onFocus: props.onFocus,
-    'aria-describedby': addIndex(props.options?.ariaDescribedby || null),
+    'aria-describedby': props.options?.ariaDescribedby
+      ? addIndex(props.options?.ariaDescribedby || null)
+      : `${props.id}-error-message`,
```

Comment:
With the ternary we should be able to simplify the argument:

```suggestion
    'aria-describedby': props.options?.ariaDescribedby
      ? addIndex(props.options.ariaDescribedby)
      : `${props.id}-error-message`,
```

Also, how do we know that the absence of the `ariaDescribedBy` option in props means that there's an error?

Reply:
@bkjohnson I updated the PR with the simplified code.

As for the error. It may not exist at the time the field is being read by a screen reader, in which case the aria-describedby does nothing.

But, should an error be generated by bad information entered, or leaving a required field blank. It'll point to the new error displayed. The error message logic and the aria-describedby aren't inherently connected unfortunately so it makes it really difficult to add/remove the aria-describedby dynamically.

In any case, having it always set has no negative repercussions.

Reply:
Interesting. If there's no downside I'm fine with having it always set, but can you add a comment in the code that explains the situation a bit? And also would it be possible to have a test check for the appropriate `aria-describeddy` based on the presence of an error?

Reply:
I've added the comment to explain the aria-describedby.
I don't believe this text widget has an associated test file that I can see

Reply:
@bkjohnson As an update to my last comment. There IS a test file. I forgot. A test for the aria-describedby labelling already exists and this change does not break the test